assets/js/index.js

```diff
@@ -1,6 +1,7 @@
 import * as govukFrontend from 'govuk-frontend'
 import * as mojFrontend from '@ministryofjustice/frontend'
-import './steps.js'
+import './autocomplete.js'
+import './datepicker.js'
```

Comment:
instead of making our own copy I think we could do `import 'hmpps-court-cases-release-dates-design/hmpps/assets/javascript/datepicker.js'` - can you try?

Reply:
It was like this only but its suggestion from Tom

Reply:
Hm OK maybe let's pick it up with @Thomas-Geraghty when he's back. I'll resolve for now though.